angular/build/config/vendor.yml

```diff
@@ -0,0 +1,43 @@
+---
```

Comment:
I am raising an eyebrow at the directory. I would have assumed that everything in build/ could be `rm -rf`'ed